onprc_ehr/src/org/labkey/onprc_ehr/history/ONPRCUrinalysisLabworkType.java

```diff
@@ -18,17 +18,14 @@
 import org.labkey.api.ehr.history.SortingLabworkType;
 import org.labkey.api.module.Module;
 
-/**
- * User: bimber
- * Date: 3/6/13
- * Time: 12:27 PM
- */
+//Modified: 10-14-2022 R.Blasa
 public class ONPRCUrinalysisLabworkType extends SortingLabworkType
 {
     public ONPRCUrinalysisLabworkType(Module module)
     {
         super("Urinalysis", "study", "Urinalysis Results", "Urinalysis", module);
         _resultField = null;
         _qualResultField = "results";
+        _testIDfieldremark = "remark";
```

Comment:
You shouldn't need this, there's already a _remarkField you can use instead of _testIDfieldremark

Reply:
I created _testIDFieldremark.  You meant that I should use a_remarkField.  Is that correct?

Reply:
Yes, both are just string variables for the word "remark" and they never change.  It's not holding any data value, just telling what column name to look at.

If you prefer to use a second variable for "remark" (_testIDfieldremark) that's fine it won't break anything logically but it should be defined here in this file, not in ehr. The code I provided in the next comment uses _remarkField, you will need to replace it there if using _testIDfieldremark.